src/lib.rs

```diff
@@ -296,6 +289,13 @@ pub fn store_path<T: Serialize>(path: impl AsRef<Path>, cfg: T) -> Result<(), Co
         s = serde_yaml::to_string(&cfg).map_err(ConfyError::SerializeYamlError)?;
     }
 
+    let mut f = OpenOptions::new()
+        .write(true)
+        .create(true)
+        .truncate(true)
+        .open(path)
+        .map_err(ConfyError::OpenConfigurationFileError)?;
+
```

Comment:
Why not chaining further?

```rust
    ...
    .open(path)
    .map_err(ConfyError::OpenConfigurationFileError)?
    .write_all(s.as_bytes())
    .map(|_| ())
    .map_err(ConfyError::WriteConfigurationFileError)
}
```

Reply:
I'm occupied in other code right now, but personally I don't like deep method chaining.

In the original code, `OpenOptions::new()` acts as "open options", and `open()` turns it into a file. If I chain the file write as well, then `write_all()` turns it into a byte count written, and `.map(|_| ())` discards the byte count written. I feel that's too many things to keep track of in one expression.

When I have the time and energy, I might try it out, assemble the whole method chain together and then see how I like it.

Reply:
Should I use:

```rs
    f.write_all(s.as_bytes())
        .map_err(ConfyError::WriteConfigurationFileError)?;
    Ok(())
```

or:

```rs
    f.write_all(s.as_bytes())
        .map_err(ConfyError::WriteConfigurationFileError)
```